onadata/apps/logger/data_migration/heal_data.py

```diff
@@ -0,0 +1,17 @@
+from onadata.apps.logger.models.instance import save_survey
+
+from .surveytree import SurveyTree
+
+
+def set_proper_root_node(xform, survey_tree):
+    new_root_id = xform.id_string
+    survey_tree.change_field_tag(survey_tree.root, new_root_id)
+    survey_tree.change_field_attrib(survey_tree.root, 'id', new_root_id)
+
+
+def set_proper_root_node_for_all_surveys_in_form(xform):
+    for survey in xform.instances.iterator():
+        survey_tree = SurveyTree(survey)
+        set_proper_root_node(xform, survey_tree)
+        survey.xml = survey_tree.to_string()
+        save_survey(survey)
```

Comment:
What is the exact purpose of this function (it's not used anywhere)? If fixing broken forms in shell - do we expect to use it often? Because otherwise it might be just dead code.

Reply:
Yes, i wrote it mostly as a helper function with intent to use it in shell. Do we expect to use it often? Well, at most once per each form, basically for each form that has issues with editing old records. Do you think that I should remove it?

Reply:
Well, if it's for the one time migration we should either do it manually or do it once automatically by making a migration file. Keeping dead code that we don't expect to use in the future might be misleading

Reply:
Ok, then we don't need this file at all, moved `set_proper_root_node` to data migrator